src/com/google/enterprise/adaptor/documentum/DocumentumAdaptor.java

```diff
@@ -1435,15 +1434,15 @@ public void getDocContent(Request req, Response resp) throws IOException {
         return;
       }
 
-      IDfPersistentObject dmPersObj;
+      IDfSysObject sysObject;
       if (path.matches(".*:\\p{XDigit}{16}")) {
         String chronicleId = path.substring(path.length() - 16);
         logger.log(Level.FINER, "Chronicle ID: {0}", chronicleId);
-        dmPersObj = dmSession.getObjectByQualification(
+        sysObject = (IDfSysObject) dmSession.getObjectByQualification(
```

Comment:
Aren't we using "dm" prefix to Documentum object types? dmSession, dmObjId, dmSysObj etc. There are some inconstancies though, query instead of dmQuery, type instead of dmType. I am fine either way, as long as we are consistent.